tools/cosmovisor/cmd/cosmovisor/init.go

```diff
@@ -6,22 +6,23 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"time"
 
 	"github.com/spf13/cobra"
 
 	"cosmossdk.io/log"
+	"cosmossdk.io/x/upgrade/plan"
+
 	"cosmossdk.io/tools/cosmovisor"
 	cverrors "cosmossdk.io/tools/cosmovisor/errors"
-	"cosmossdk.io/x/upgrade/plan"
 )
 
 var initCmd = &cobra.Command{
 	Use:   "init <path to executable>",
 	Short: "Initialize a cosmovisor daemon home directory.",
 	Args:  cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		logger := cmd.Context().Value(log.ContextKey).(log.Logger)
-		return InitializeCosmovisor(logger, args)
+		return InitializeCosmovisor(nil, args)
```

Comment:
I do not think it is used anywhere, so I would simply remove the logger from the argument instead of passing nil.